CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ This is the log of notable changes to EAS CLI and related packages.
 - Update `eas-cli` and `@expo/eas-json` dependencies. ([#2176](https://github.com/expo/eas-cli/pull/2176) by [@szdziedzic](https://github.com/szdziedzic))
 - Update `eas.schema.json` to after adding Xcode 15.2 image. ([#2184](https://github.com/expo/eas-cli/pull/2184) by [@szdziedzic](https://github.com/szdziedzic))
 - Upgrade packages from the expo/expo repo. ([#2145](https://github.com/expo/eas-cli/pull/2145) by [@wschurman](https://github.com/wschurman))
+- Remove support for classic updates release channel in 50+. ([#2189](https://github.com/expo/eas-cli/pull/2189) by [@wschurman](https://github.com/wschurman))
```

Comment:
If you merge it without rebasing, this changelog entry will be added under the version `6.1.0` section, and not under the not yet released `main` branch section. So please rebase this PR 🙏